visualization.shadertoy/changelog.txt

```diff
@@ -1,3 +1,9 @@
+[B]20.1.0[/B]
+- Add new feature to allow read of available presets from a external file in JSON format
+  - There one as standard with 'presets.json' on addon
+  - Further can user set within settings to a own file
+  - Also to help about is within settings a button added to open text window with description about
+
```

Comment:
- Add new custom shader list feature:
  - Move existing shader list from add-on code to the default 'presets.json' file
  - Allow users to configure a path to their own custom preset file
  - Add help button with instructions on preset file JSON formatting

Reply:
Sounds much better, has changed too.